test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.py

```diff
@@ -0,0 +1,272 @@
+            )
+
+        # For FromVolume resume policy PVC should not be deleted.
+        if experiment.spec.resume_policy == "FromVolume":
+            PVCs = client.CoreV1Api().list_namespaced_persistent_volume_claim(
+                exp_namespace
+            )
+            is_deleted = 1
+            for i in PVCs.items:
+                if i.metadata.name == resource_name:
+                    is_deleted = 0
+            if is_deleted == 1:
+                raise Exception(
+                    "PVC is deleted for FromVolume resume policy. "
+                    f"Alive PVCs: {[i.metadata.name for i in PVCs.items]}."
+                )
+
+
+def run_e2e_experiment(
+    katib_client: KatibClient,
+    experiment: models.V1beta1Experiment,
+    exp_name: str,
+    exp_namespace: str,
+):
+
+    # Create Katib Experiment and wait until it is finished.
+    print(
+        "\n\n\nCreating Experiment: {}/{} with MaxTrialCount: {}, ParallelTrialCount: {}".format(
+            exp_namespace,
+            exp_name,
+            experiment.spec.max_trial_count,
+            experiment.spec.parallel_trial_count,
+        )
+    )
+
+    # Wait until Experiment reaches Succeeded condition.
+    katib_client.create_experiment(experiment, exp_namespace)
+    experiment = katib_client.wait_for_experiment_condition(
+        exp_name, exp_namespace, timeout=EXPERIMENT_TIMEOUT
+    )
+
+    # Test resume feature for "FromVolume" and "LongRunning" Experiments.
+    # TODO (andreyvelich): Once we change the default resume policy to "Never",
+    # also test restart for "LongRunning" Experiment here instead of "random".
+    # Ref: https://github.com/kubeflow/katib/issues/2055
+    if exp_name == "from-volume-resume" or exp_name == "random":
+        max_trial_count = experiment.spec.max_trial_count + 1
+        parallel_trial_count = experiment.spec.parallel_trial_count + 1
+        print(
+            f"Restarting Experiment {exp_namespace}/{exp_name} "
+            f"with MaxTrialCount: {max_trial_count} and ParallelTrialCount: {parallel_trial_count}"
+        )
+
+        # Modify Experiment budget.
+        katib_client.edit_experiment_budget(
+            exp_name, exp_namespace, max_trial_count, parallel_trial_count
+        )
+        # Wait until Experiment is Restarted.
+        katib_client.wait_for_experiment_condition(
+            exp_name,
+            exp_namespace,
+            constants.EXPERIMENT_CONDITION_RESTARTING,
+            EXPERIMENT_TIMEOUT,
+        )
+        # Wait until Experiment is Succeeded.
+        experiment = katib_client.wait_for_experiment_condition(
+            exp_name, exp_namespace, timeout=EXPERIMENT_TIMEOUT
+        )
+
+    # Verify the Experiment results.
+    verify_experiment_results(katib_client, experiment, exp_name, exp_namespace)
+
+    # Describe the Experiment and Suggestion.
+    print(os.popen(f"kubectl describe experiment {exp_name} -n {exp_namespace}").read())
```

Comment:
Can we add `verbose` as an optional parameter that can be enabled/disabled to show verbose logs? (Enabled by default) 

This will help in tests like conformance tests where we are only interested in final experiment success/failure

Reply:
Sure, let me enable it.

Reply:
@johnugeorge I enabled logging for Katib E2Es, what do you think about it ?